tools/matchms/matchms_convert.xml

```diff
@@ -0,0 +1,64 @@
+<tool id="matchms_convert" name="matchms convert" version="@TOOL_VERSION@+galaxy0" python_template_version="3.7">
```

Comment:
remove template_version but include profile

Reply:
resolved in https://github.com/RECETOX/galaxytools/pull/339/commits/3cad13a675a84c166bc3c6f29d31cde561822c7b